src/extensions/otel_viewer_extension/Dashboard.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Diagnostics;
+using Azure.AI.Details.Common.CLI;
+
+namespace Azure.AI.Details.Common.CLI.Extensions.Otel
+{
+    public class Dashboard
+    {
+        // Execute a command in the command prompt and return the output
+        private static string ExecuteCommand(string command)
+        {
+            ProcessStartInfo processStartInfo = new ProcessStartInfo
+            {
+                FileName = "cmd.exe",
```

Comment:
there are some other functions/helpers in common library that can help launch things more appropriately, to ensure that it works both on Windows, Linux, and Mac. Some are in `process_helpers.cs`.

Reply:
i mention this here as the "cmd.exe" will only work on Windows.